services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -745,6 +765,57 @@ class FlowCrudSpec extends HealthCheckSpecification {
         }
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2625")
+    def "System recreates excess meter when flow is created with the same meterId"() {
+        given: "A Noviflow switch"
+        def sw = topology.activeSwitches.find { it.noviflow }
+
+        and: "Create excess meters on the given switch"
+        def fakeBandwidth = 333
+        def amountOfExcessMeters = 10
+        amountOfExcessMeters.times {
+            def producer = new KafkaProducer(producerProps)
+            //pick a meter id which is not yet used on src switch
+            def excessMeterId = ((MIN_FLOW_METER_ID..100) - northbound.getAllMeters(sw.dpId)
+                    .meterEntries*.meterId).first()
+            producer.send(new ProducerRecord(flowTopic, sw.dpId.toString(), buildMessage(
+                    new InstallIngressFlow(UUID.randomUUID(), NON_EXISTENT_FLOW_ID, null, sw.dpId,
+                            5, 6, 5, excessMeterId, FlowEncapsulationType.TRANSIT_VLAN,
+                            OutputVlanType.REPLACE, fakeBandwidth, excessMeterId, sw.dpId)).toJson()))
+
+            producer && producer.close()
+        }
+
+        assert northbound.validateSwitch(sw.dpId).meters.excess.size() == amountOfExcessMeters
+
+        when: "Create several flows"
+        List<String> flows = []
+        def amountOfFlows = 3
+        def allowedPorts = topology.getAllowedPortsForSwitch(sw)
+        amountOfFlows.times {
+            def flow = flowHelper.singleSwitchFlow(sw)
+            flow.source.portNumber = allowedPorts[-1]
+            flow.destination.portNumber = allowedPorts[-2]
+            northbound.addFlow(flow)
+            flows << flow.id
+        }
+
+        then: "Some excess meters should be recreated"
```

Comment:
I don't quite understand the meaning of this

Reply:
renamed
and comment was left